src/ext/oblivc/obliv_bits.c

```diff
@@ -1842,6 +681,7 @@ void feedOblivInputs(OblivInputs* spec, size_t count, int party)
     { __obliv_c__##ot rv; \
       OblivInputs spec; \
       setupObliv##tname(&spec,&rv,v); \
+      spec.src_f = v; \
```

Comment:
Ok, I see what you mean :/

In my testing I was not calling feedOblivInt and instead just doing a direct assignment. I forgot that that would not work because that calls setup without going through the feed function. I really apologize for being stubborn and for the messiness! You are correct, the following fix seems to work (moving the ```spec.src_f = v``` to the setup function):
```
void setupOblivFloat(OblivInputs* spec, __obliv_c__float* dest, float v)
  { 
    __obliv_c__setupOblivBits(spec,dest->bits,v,__bitsize(v));
    spec->src_f = v;
  }
```

I will include this as part of the other improvements I am doing to this pull request unless you see any problems with that.

Thanks :)